### ISSUE TITLE
Muting and deafening list the invoker as the target

### ISSUE THREAD

**Issue by endreman0** [FRCDiscord/Dozer#60]
Tested on development fork (prefix `e&`) with no relevant changes. Accounts involved: endreman0#9853 (caller), endreman1#1276 (target), Dozer-endreman0#6997 (bot)
```
      endreman0: e&deafen endreman1
Dozer-endreman0: endreman0#9853 has deafened endreman0#9853 because No reason provided
      endreman0: e&undeafen endreman1
Dozer-endreman0: endreman0#9853 has undeafened endreman1#1276 because No reason provided
Dozer-endreman0: Please configure modlog channel to enable modlog functionality
      endreman0: e&mute endreman1
Dozer-endreman0: endreman0#9853 has muted endreman0#9853 because No reason provided
Dozer-endreman0: Please configure modlog channel to enable modlog functionality
      endreman0: e&unmute endreman1
Dozer-endreman0: endreman0#9853 has unmuted endreman1#1276 because No reason provided
Dozer-endreman0: Please configure modlog channel to enable modlog functionality
```
Other examples visible in "FIRST® Robotics Competition" guild, channel #spam180.

**Comment by tweirtx** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Moderation display error](https://trello.com/c/u3zREtKu/60-moderation-display-error)

**Comment by tweirtx** [comment]
Fixed in PR #63
